src/Summoner/Template.hs

```diff
@@ -267,14 +267,24 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
           "https://img.shields.io/hackage/v/" <> repo <> ".svg"
         hackageLink :: Text =
           "https://hackage.haskell.org/package/" <> repo
+
         travisShield :: Text =
           "https://secure.travis-ci.org/" <> owner <> "/" <> repo <> ".svg"
         travisLink :: Text =
           "https://travis-ci.org/" <> owner <> "/" <> repo
+        travisBadge :: Text = if travis
+            then [text|[![Build status](${travisShield})](${travisLink})|]
+            else ""
```

Comment:
I think you can use `emptyIfNot` function for such cases:

https://github.com/kowainik/summoner/blob/108f79c223046f2b8b8591b3e6e16cf95eada736/src/Summoner/Template.hs#L27-L28